rules/react.json

```diff
@@ -0,0 +1,174 @@
+{
+  "rules": {
+    "react/display-name": [
+      "off",
+      {
+        "ignoreTranspilerName": false
+      }
+    ],
+    "react/forbid-prop-types": [
+      "error",
+      {
+        "forbid": [
+          "any",
+          "array",
+          "object"
+        ],
+        "checkContextTypes": true,
+        "checkChildContextTypes": true
+      }
+    ],
+    "react/jsx-boolean-value": [
+      "error",
+      "never",
+      {
+        "always": []
+      }
+    ],
+    "react/jsx-max-props-per-line": [
+      "error",
+      {
+        "maximum": 1,
+        "when": "multiline"
+      }
+    ],
+    "react/jsx-no-bind": "error",
+    "react/jsx-no-duplicate-props": [
+      "error",
+      {
+        "ignoreCase": true
+      }
+    ],
+    "react/jsx-pascal-case": [
+      "error",
+      {
+        "allowAllCaps": true,
+        "ignore": []
+      }
+    ],
+    "react/jsx-uses-react": [
+      "error"
+    ],
+    "react/no-danger": "warn",
+    "react/no-did-update-set-state": "error",
+    "react/no-will-update-set-state": "error",
+    "react/no-string-refs": "error",
+    "react/prefer-es6-class": [
+      "error",
+      "always"
+    ],
+    "react/prefer-stateless-function": [
+      "error",
+      {
+        "ignorePureComponents": true
+      }
+    ],
+    "react/prop-types": [
+      "error",
```

Comment:
Думаю это правило нужно отключить, т.к. из-за него не существующая ошибка, свойства TS Проверит
`'children' is missing in props validationeslintreact/prop-types`

Reply:
Насколько я понимаю, проблема уже не актуальна после обновления eslint-plugin-react https://github.com/Studio-Yandex-Practicum/lubimovka_frontend/commit/434516bb61c883a2275d28928706f53a43110f35

Reply:
Мне кажется проблема актуальна. 
В документации написано `Since 2.0.0 children is no longer ignored for props validation.`. Или я что-то не понимаю.
Точно не понимаю зачем c ts нужно это правило.